doc/sphinx/source/users-guide/locality/compilingAndExecutingMultiLocalePrograms.rst

```diff
@@ -0,0 +1,75 @@
+:title: Users Guide: Compiling and Executing Multi-Locale Programs
+
+Compiling and Executing Multi-Locale Programs
+=============================================
+
+Selecting a Communication Mechanism
+-----------------------------------
+
+All multi-locale Chapel programs are implemented using runtime
+communication routines which transparently move data and tasks between
+the locales on which the program is running.  The choice of
+communication runtime is made via the ``--comm`` compiler flag (or
+equivalently, the ``$CHPL_COMM`` environment variable).  Portable
+options for communication include:
+
+* ``none``: specifies that no communication should be performed,
+  restricting the program's execution to a single locale.
+
+* ``gasnet``: specifies that communication should be implemented using
+  the open-source GASNet library developed at Berkeley.
+
+When using the Chapel module on Cray systems, a third option is also
+available:
+
+* ``ugni``: specifies that communication should be implemented using
+  Cray's uGNI interface.
+
+In order to use any of these options, the Chapel runtime libraries
+must have been built with the specified communication layer enabled.
```

Comment:
I recall we were trying to get away from the terminologies of "communication layer" and "tasking layer" when working on the man-pages - referring to them as "communication implementation" and "tasking implementation" instead.

(This comment applies to all of the occurrences: `grep -R "layer" *`)